cmd/soroban-rpc/internal/db/util.go

```diff
@@ -0,0 +1,107 @@
+package db
```

Comment:
This code only seems to be used in tests, it shouldn't be in the `db` package (since it gets incorporated into production code, causing bloat). Please place it in a `_test.go` file or in a package only used by tests.

Reply:
I wanted to remove duplicated code since txMeta function is used in multiple places. However, I cant move it to infrastructure file since it creates an import cycle with the db package. I have reverted the util.go file creation and added the txMeta implementation to the infrastructure util.go file for now.

Reply:
Sounds good